scripts/commands/update/update.js

```diff
@@ -0,0 +1,72 @@
+const nextTokens = {
+  core: require("momentum-abstract/color/core.json"),
+};
+
+const prevTokens = {
+  decorative: require("../../../core/color/decorative.json"),
+  functional: require("../../../core/color/functional.json"),
+};
+
+const utils = require("../../utils");
+const common = require("../../common");
+
+const ColorToken = require("../../models/color-token");
+
+/**
+ * Update all tokens within this project.
+ */
+const update = () => {
+  const next = {
+    functional: {
+      color: new ColorToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        category: ColorToken.CONSTANTS.CATEGORIES.COLOR,
+        data: nextTokens.core["core color"],
+        name: "functional",
+      }).normalize(),
```

Comment:
why is the normalize necessary here?

Reply:
`new ColorToken({ /* ... */ }).normalize()` converts the token immediately to `standard` format, which follows the formatting of the local token files. In some cases [like with gradients], tokens need additional steps if they aren't standardized/normalized, which is why this isn't amended as a part of the constructor flow.

Reply:
sounds good to me - i think its worth indicating that somehow (we don't have TS yet, which would have been helpful here, so a comment or something I guess would be good) :)

Reply:
The JSDoc indicates this currently:

```js
  /**
   * Normalizes this ColorToken into the project's Standard format.
   *
   * @returns {this} - This color token.
   */
  normalize() {
    return this.updateFormat({ format: Token.CONSTANTS.TOKEN_FORMATS.STANDARD });
  }
  ```
  
  Let me know if this isn't enough.
  
  >we don't have TS yet, which would have been helpful here, so a comment or something I guess would be good
  
  This is a very valid point, TypeScript would have provided a resolution to this comment.